x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/use_response_actions_log_table.tsx

```diff
@@ -219,12 +234,13 @@ export const useResponseActionsLogTable = ({
             <EuiFlexGroup
               data-test-subj={getTestId('details-tray')}
               direction="column"
-              style={{ maxHeight: 270, overflowY: 'auto' }}
+              // TODO check if this maxHeight is really necessary
+              // style={{ maxHeight: 270, overflowY: 'auto' }}
```

Comment:
So we have set the `max-height` so that the expanded tray height is consistent across each table item. The expanded sections are scrollable anyway. The table content for osquery output should be scrollable as well.

Reply:
Thanks @ashokaditya for a quick response 👍  That Osquery table requires a bit more of space than 270, otherwise it is hardly readable. However, it's just a draft of functionality - let's get some feedback from UX and find out a solution to nicely present all of them :)

Reply:
Yes, I agree.  The osquery table output could use some more height. It would be easy to just increase the height but that would mean in all other action outputs we would see a lot of vertical white space. We definitely need some design input here.